tests/test_main.py

```diff
@@ -35,9 +37,11 @@ def get_matching_helper(request, context):
         elif wsfunction == moodle.MOODLE_FUNC_GET_SELF_ENROLMENT_METHODS:
             return [{'id': 2}]
         elif wsfunction == moodle.MOODLE_FUNC_CORE_ENROL_GET_ENROLLED_USERS:
-            return {'courseid': 21, 'users': {'id': 3}}
+            return [{'id': 2}, {'id': 3}]
```

Comment:
Are we changing this mock response because it was a test bug before?

Reply:
Yes - this new return better reflects the structure of the moodle function. Previous tests just checked that ^ this old object was returned. But for new test we need to modify this structure after it's returned so it needs to have at least an `id` value and be a list of objects.